intezer_analyze_cli/commands.py

```diff
@@ -277,3 +278,104 @@ def index_directory_command(directory_path: str,
                 except Exception:
                     click.echo('error occurred during indexing of {}'.format(index_result['file_name']))
                     progressbar.update(1)
+
+
+def upload_offline_endpoint_scan(offline_scan_directory: str, force: bool = False):
+    try:
+        if not force and _was_directory_already_sent(offline_scan_directory):
+            raise click.Abort()
+        endpoint_analysis = EndpointAnalysis(offline_scan_directory=offline_scan_directory)
+        endpoint_analysis.send(wait=False)
+        if not endpoint_analysis.analysis_id:
+            raise RuntimeError('Error encountered while sending offline scan, server did not return analysis id')
+        _create_analysis_id_file(offline_scan_directory, endpoint_analysis.analysis_id)
+
+        if default_config.is_cloud:
+            click.echo(
+                'Analysis created. In order to check its result, go to: {}/{}'.format(
+                    default_config.endpoint_analyses_url,
+                    endpoint_analysis.analysis_id))
+        else:
+            click.echo('Analysis created. In order to check its result go to Intezer Analyze history page')
+    except sdk_errors.IntezerError as e:
+        click.echo('Analyze error: {}'.format(e))
+
+
+def upload_multiple_offline_endpoint_scans(offline_scans_root_directory: str,
+                                           force: bool = False):
+    success_number = 0
+    failed_number = 0
+
+    directories = [d for d in os.listdir(offline_scans_root_directory) if
+                   os.path.isdir(os.path.join(offline_scans_root_directory, d)) and
+                   not is_hidden(os.path.join(offline_scans_root_directory, d))]
+
+    if 'files' not in directories:
+        click.echo('Directory "files" is missing')
+        raise click.Abort()
+    directories.remove('files')
+
+    if 'fileless' not in directories:
+        click.echo('Directory "fileless" is missing')
+        raise click.Abort()
+    directories.remove('fileless')
+
+    if 'memory_modules' not in directories:
+        click.echo('Directory "memory_modules" is missing')
+        raise click.Abort()
+    directories.remove('memory_modules')
+
+    with click.progressbar(length=len(directories),
+                           label='Sending offline endpoint scans for analysis',
+                           show_pos=True) as progressbar:
+        for scan_dir in directories:
+            offline_scan_directory = os.path.join(offline_scans_root_directory, scan_dir)
```

Comment:
You have duplicate of code, why not to use `upload_offline_endpoint_scan`?
It makes the function huge